src/main/java/tempest/GUI/ManageModulesPage.java

```diff
@@ -0,0 +1,93 @@
+package tempest.GUI;
+
+import java.awt.event.ActionEvent;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JPanel;
+
+import tempest.GUI.components.BackButton;
+import tempest.GUI.components.ModuleDropDown;
+
+public class ManageModulesPage extends Page {
+    private final GUIManager manager;
+    private final ModuleDropDown moduleDropDown = new ModuleDropDown();
+
+    private JPanel manageModulesPanel;
+    private JPanel pageSwapPanel;
+    private JPanel backPanel;
+
+    private JButton addModuleButton;
+    private JButton deleteModuleButton;
+    private BackButton backButton;
+    private JComboBox<Object> dropDown;
+
+    public ManageModulesPage(GUIManager guiManager) {
+        this.manager = guiManager;
+    }
+
+    @Override
+    public String getName() {
+        return "manageModulesPage";
+    }
+
+    public JPanel getPanel() {
+        manageModulesPanel = new JPanel();
+        pageSwapPanel = new JPanel();
+        backPanel = new JPanel();
+
+        dropDown = moduleDropDown.getModuleDropDown();
+
+        addModuleButton = new JButton("Add a module");
+        deleteModuleButton = new JButton("Delete a module");
```

Comment:
If you look at the `homePage` you'll see we are now using the `LinkButton `component for buttons taking the user to other pages.